src/uwtools/tests/config/formats/test_base.py

```diff
@@ -127,6 +128,18 @@ def test_dereference(caplog, config):
     assert config == {"foo": 88, "a": 77, "b": {"c": 66}, "d": "{{ X }}"}
 
 
+@pytest.mark.parametrize("fmt2", [FORMAT.ini, FORMAT.nml, FORMAT.sh])
+def test_invalid_config(fmt2, tmp_path):
+    """
+    Test that invalid config files will error when attempting to dump.
+    """
+    fmt1 = FORMAT.yaml
+    outfile = tmp_path / f"test_{fmt1.lower()}to{fmt2.lower()}_dump.{fmt2}"
+    cfgin = tools.format_to_config(fmt1)(fixture_path(f"hello_workflow.{fmt1}"))
+    with raises(UWConfigError):
+        tools.format_to_config(fmt2).dump_dict(path=outfile, cfg=cfgin.data)
```

Comment:
Does it make sense to capture that the right log message shows up here?

Reply:
Yep, got to change `raises` to matching the log message

Reply:
Got it, kept both tests too